src/systems/fem_context.C

```diff
@@ -1532,6 +1554,7 @@ void FEMContext::pre_fe_reinit(const System &sys, const Elem *e)
   if (sys.use_fixed_solution)
     this->get_elem_fixed_solution().resize(n_dofs);
   this->get_elem_solution_rate().resize(n_dofs);
+  this->get_elem_solution_accel().resize(n_dofs);
```

Comment:
Huh.  No, they aren't, and yes, that sounds like a good idea.  We'll eventually need to add a just-build-everything codepath for the !LIBMESH_HAVE_RTTI case, but that's not an important configuration and I don't think it's currently working either.